autoql_python_backend/data_alerts/routes.py

```diff
@@ -0,0 +1,11 @@
+from flask import jsonify
+
+from autoql_python_backend import app
+from .models import DataAlertNotification
+
+
+@app.route('/notifications/<string:notification_id>')
```

Comment:
is default method GET?

Reply:
Yeah, but I should define it for consistency.